app/src/main/java/com/jalasoft/routesapp/data/model/remote/Line.kt

```diff
@@ -0,0 +1,7 @@
+package com.jalasoft.routesapp.data.model.remote
+
+data class Line(
+    val route: List<List<String>>,
```

Comment:
The list from the lines are GeoPoints, consider using that class or Location class